pyproject.toml

```
@@ -50,5 +55,25 @@ local_scheme = "no-local-version"
 
 [tool.black]
```

Possibly we could enable this setting for black. I don't much like the magic trailing comma, and I think most of the current black maintainers don't really like it much either. But I also don't care that much, if you disagree :)

```suggestion
[tool.black]
skip-magic-trailing-comma = true
```

---

I don't mind too much. This is the only change it would make:

```diff
diff --git a/sphinxlint/cli.py b/sphinxlint/cli.py
index c0a5638..0cab6d4 100644
--- a/sphinxlint/cli.py
+++ b/sphinxlint/cli.py
@@ -76,11 +76,7 @@ def parse_args(argv=None):
         help="verbose (print all checked file names)",
     )
     parser.add_argument(
-        "-i",
-        "--ignore",
-        action="append",
-        help="ignore subdir or file path",
-        default=[],
+        "-i", "--ignore", action="append", help="ignore subdir or file path", default=[]
     )
     parser.add_argument(
         "-d",
```

I can see there could be value keeping it, if we wanted to make sure each `parser.add_argument` block had each arg on its own line. 🤷

---

I'd personally vote for my suggestion above^, but no strong opinion either way. Feel free to merge!